recipe/meta.yaml

```
@@ -28,6 +29,7 @@ test:
   imports:
     - napari_matplotlib
   commands:
+    - "python -c 'from napari_matplotlib._version import version;print(version)'"
```

```suggestion
    - "python -c 'from napari_matplotlib import __version__;print(__version__)'"
```
So it's testing public API?

---

Yep that also works, thanks :) 

When using `pip list` I was getting a 0.0.0 version which is affecting another pip check on a different PR that depended on this one.

---

See https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=518824&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=2651

I missed that :)